ios/Sources/AboutFeature/StaffCell.swift

```diff
@@ -1,17 +1,20 @@
 import Component
+import Model
 import Styleguide
 import SwiftUI
 
 public struct StaffCell: View {
     private let staff: Staff
+    private let onTap: (Staff) -> Void
```

Comment:
I think it's better to unify the name of action. 
In current app we name `tapAction`. ref: https://github.com/DroidKaigi/conference-app-2021/blob/b82b2bb4a64a13400c4754abbb0b6c1f4cecbb4d/ios/Sources/Component/Card/SmallCard.swift#L17